apps/mobile/src/components/add-wallet/add-wallet-sheet.layout.tsx

```diff
@@ -87,6 +63,42 @@ export function AddWalletSheetLayout({
     marginBottom: interpolate(animatedIndex.value, [-1, 0], [200, 0], Extrapolation.CLAMP),
   }));
 
+  function getUnavailableFeatures(theme: Theme) {
```

Comment:
Maybe this exist in it's own file to format the data then pass in a simpler way? 

So this could accept `featureEntries` where `featureEntries = Object.entries(getUnavailableFeatures(theme))`

The `.layout` file should just be UI formatting